lib/controllers/messages.ts

```diff
@@ -606,15 +615,31 @@ export function GetMessageHandler(
         }|${errorOrNotificationStatuses.value.message}`
       );
     }
-
     const notificationStatuses = errorOrNotificationStatuses.value;
 
-    const messageStatus: MessageResponseWithContent = {
+    const errorOrMaybeMessageStatus = await messageStatusModel.findOneByMessageId(
+      retrievedMessage.id
+    );
+
+    if (isLeft(errorOrMaybeMessageStatus)) {
+      return ResponseErrorInternal(
+        `Error retrieving MessageStatus: ${
+          errorOrMaybeMessageStatus.value.code
+        }|${errorOrMaybeMessageStatus.value.body}`
+      );
+    }
+    const maybeMessageStatus = errorOrMaybeMessageStatus.value;
+
+    const returnedMessage: MessageResponseWithContent = {
       message,
-      notification: notificationStatuses.toUndefined()
+      notification: notificationStatuses.toUndefined(),
+      // we do not return the status date-time
+      status: maybeMessageStatus
+        .map(messageStatus => messageStatus.status)
+        .getOrElse(MessageStatusValueEnum.ACCEPTED)
```

Comment:
a brief explanation of this behavior would be useful here (e.g. when the message has been stored but a `MessageStatus` does not exist, the message is considered to be in the `ACCEPTED` state)